File: Python/car/carAdvanced.py
from car import car

class carAdvanced(car):
    typeCarAccepted = []
    seatsMaterial   = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super.__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

File: Java/Car/CarAdvanced.java
package Java.Car;

import java.util.ArrayList;
import java.util.Map;

import Java.AccountType.Driver;

public class CarAdvanced extends Car{
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public CarAdvanced(String license, Driver driver, Map<String, Map<String, Integer>> typeCarAccepted, ArrayList<String> seatsMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }

    public CarAdvanced(String license, Driver driver) {
        super(license, driver);
    }

}
